src/Bicep.Wasm/Bicep.Wasm.csproj

```diff
@@ -7,6 +7,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly" Version="6.0.9" />
     <PackageReference Include="System.Net.Http.Json" Version="6.0.0" />
+    <PackageReference Include="System.IO.Abstractions.TestingHelpers" Version="17.2.3" />
```

Comment:
Should this have a `PrivateAssets="all"` attribute?

Reply:
Here I think it makes sense to keep the default behavior - this project isn't being packaged